### PR TITLE
[MINVOKER-306] - Fixed bug if only exclude-patterns specified

### PR DESCRIPTION
**Pull request by bvfalcon**
In addition to [MINVOKER-140](https://issues.apache.org/jira/browse/MINVOKER-140): exclude patterns works only if one or more include pattern(s) exists. This is a feature of [plexus-utils Component DirectoryScanner](https://github.com/codehaus-plexus/plexus-utils/blob/3896620b4fbf031b1b02b4d46b328c5839a231da/src/main/java/org/codehaus/plexus/util/DirectoryScanner.java#L447-L449). 

So, in our case command line parameter <code>-Dinvoker.test=!testX</code> (and configuration parameter [invokerTest](https://maven.apache.org/plugins/maven-invoker-plugin/run-mojo.html#invokerTest) ) works absolutely different way: **all projects** (not only testX) will be skipped. We must use <code>-Dinvoker.test=\*,!testX</code> or <code>-Dinvoker.test=!testX,\*</code> as workarround.

In this commit I add forced additional include pattern "*" as default include pattern for cases, when invoker.test contains only exclude-patterns. User still use <code>-Dinvoker.test=!testX</code> and all will work correctly for him. The change is full back-compatibile with current plugin settings.

Integration test invocation-cmdline-exclude is actualized, JIRA Ticket created: [MINVOKER-306](https://issues.apache.org/jira/projects/MINVOKER/issues/MINVOKER-306).

Following this checklist to help us incorporate your contribution quickly and easily:

- [x]  Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MINVOKER) filed
      for the change (usually before you start working on it).  Trivial changes like typos do not
      require a JIRA issue.  Your pull request should address just this issue, without
      pulling in other changes.

- [ ]  Each commit in the pull request should have a meaningful subject line and body.

- [x]  Format the pull request title like `[MINVOKER-XXX] - Fixes bug in ApproximateQuantiles`,
      where you replace `MINVOKER-XXX` with the appropriate JIRA issue. Best practice
      is to use the JIRA issue title in the pull request title and in the first line of the
      commit message.

- [x]  Write a pull request description that is detailed enough to understand what the pull request does, how, and why.

- [x]  Run `mvn clean verify` to make sure basic checks pass. A more thorough check will
      be performed on your pull request automatically.

- [x]  You have run the integration tests successfully (`mvn -Prun-its clean verify`).

If your pull request is about ~20 lines of code you don't need to sign an [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure please ask on the developers list.

To make clear that you license your contribution under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) you have to acknowledge this by using the following check-box.

- [x]  I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

- [x]  In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

File: src/it/invocation-cmdline-exclude/src/it/minvoker-140-executable/pom.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+  
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>1.1.1</version>
```
Comment (slawekjaranowski) [inline src/it/invocation-cmdline-exclude/src/it/minvoker-140-executable/pom.xml:38]: very old version 😄 pleas use the latest
please also update in `minvoker-140-ignorable/pom.xml`

File: src/it/invocation-cmdline-exclude/verify.bsh [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/*
```
Comment (slawekjaranowski) [inline src/it/invocation-cmdline-exclude/verify.bsh:1]: please use groovy

**Review by slawekjaranowski — CHANGES_REQUESTED** [review]
We change some of logic - please create new issue with description, examples and justifications

**Comment by bvfalcon** [comment]
> We change some of logic - please create new issue with description, examples and justifications

This PR is only bugfix, it contains nothing new. Logic is the same - "all except testX", but syntax <code>-Dinvoker.test=!testX</code> not works and syntax <code>-Dinvoker.test=\*,!testX</code> not so obvious.

**Comment by slawekjaranowski** [comment]
You are right it is a bug. 
Thanks for your efforts.

We need issues for tracking - what next release will contains.

You also remove default PR template, which contains some of formal information about contributing license - please restore at lest such information.

**Comment by bvfalcon** [comment]
Done. Jira ticket created - [MINVOKER-306](https://issues.apache.org/jira/projects/MINVOKER/issues/MINVOKER-306) and pull request description modified to standard view

**Review by slawekjaranowski — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #439

**Comment by jira-importer** [comment]
Resolve #525

**Pull request merged**